### PR TITLE
Build support for Precise and Trusty

### PR DESCRIPTION
**Pull request by daniviga**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,7 @@
+  [Matteo Nastasi]
```
Comment (nastasi-oq) [inline debian/changelog]: No Vigano item ?

File: debian/rules [hunk 2]
```diff
@@ -16,6 +16,18 @@ export DH_OPTIONS
 
 bd=openquake/bin
 
+TRUSTY_DEP = "python-amqp, python-django"
+TRUSTY_REC = "postgresql-9.3, postgresql-9.3-postgis-2.1"
+PRECISE_DEP = "python-django16"
+PRECISE_REC = "postgresql-9.1, postgresql-9.1-postgis"
+
+ifeq ($(shell lsb_release --codename --short),trusty)
+	DEPENDS = -Vdist:Depends=$(TRUSTY_DEP)
```
Comment (nastasi-oq) [inline debian/rules]: this indirection works fine with spaces inside TRUSTY_DEP var ?

File: packager.sh [hunk 3]
```diff
@@ -233,7 +235,7 @@ _pkgbuild_innervm_run () {
     ssh $lxc_ip "sudo mk-build-deps --install --tool 'apt-get -y' build-deb/debian/control"
 
     ssh $lxc_ip "cd build-deb && dpkg-buildpackage $DPBP_FLAG"
-    scp -r $lxc_ip:*.{tar.gz,deb,changes,dsc} ../
+    scp -r ${lxc_ip}:*.{tar.gz,deb,changes,dsc} ..
```
Comment (nastasi-oq) [inline packager.sh]: maybe we can add '/' again

**Comment by nastasi-oq** [comment]
LGTM

**Comment by daniviga** [comment]
LGTM too

**Pull request merged**